addon/components/connect.js

```diff
@@ -37,9 +38,15 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
           });
         }
 
-        this.actions = Object.assign({},
-          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
-        );
+        if (typeof dispatchToActions === 'function') {
+          this.actions = Object.assign({},
+            this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
+          );
+        }
+
+        if (typeof dispatchToActions === 'object') {
+          this.actions = Object.assign({}, this.actions, bindActionCreators.call(this, dispatchToActions, redux.dispatch.bind(redux)));
```

Comment:
Looking at the `bindActionCreators` implementation, it seems like it use `this`, so it seems like it might be unnecessary to call it with the current context, like:
```
bindActionCreators(dispatchToActions, redux.dispatch.bind(redux))
```